proof/invariant-abstract/AARCH64/ArchVCPU_AI.thy

```diff
@@ -9,7 +9,7 @@ theory ArchVCPU_AI
 imports AInvs
 begin
 
-context Arch begin global_naming AARCH64 (*FIXME: arch_split*)
+context Arch begin arch_global_naming (*FIXME: arch_split*)
```

Comment:
Not relevant to this PR, but is the FIXME suggesting that some of this file could be generic? Because that seems unlikely, unless we come up with a new mechanism to only be generic between certain architectures.

Reply:
I honestly have no idea. My guess is that it's an indication that a whole bunch of things were shoved in without looking too deeply... but I'm not seeing any generic stuff that should be reconsidered or exported either. I think removing the FIXME is the best idea here. Will do that for next iteration.